app/controlplane/internal/conf/controlplane/config/v1/conf.proto

```diff
@@ -135,3 +139,11 @@ message CA {
     string key_pass = 3;
   }
 }
+
+// OnboardingSpec is a configuration to automatically onboard users in organizations with specific roles
+message OnboardingSpec {
+  // Name of the organization
+  string name = 1;
```

Comment:
for next time since we are not being consistent. We could add the validations too `[(buf.validate.field).string.min_len = 1];`